src/services/call.ts

```diff
@@ -132,6 +134,33 @@ export class CallService {
         return true;
     }
 
+    public static async makeOutgoingCall(
+        user: User,
+        cursor: Cursor<RemoteData<Voximplant['Call']>>
+    ): Promise<RemoteDataResult<Voximplant['Call']>> {
+        cursor.set(loading);
+
+        try {
```

Comment:
You don't need to use 'try' block twice. Just handle all errors in `catch` block